pytorch_lightning/loggers/base.py

```diff
@@ -410,18 +410,30 @@ def close(self) -> None:
         for logger in self._logger_iterable:
             logger.close()
 
+    @staticmethod
+    def all_equal(x):
+        return all(el == x[0] for el in x)
+
```

Comment:
1. this doesn't need to be on the public API for the logger collection interface. we could define this as a helper outside the class and use it inside the `name` and `version` properties
2. please add typehints for new new code: https://github.com/PyTorchLightning/pytorch-lightning/issues/5023

i'm unclear what `x` is supposed to be and whether `x[0]` is a valid index access